eng/common/scripts/Add-Issue-Labels.ps1

```diff
@@ -0,0 +1,28 @@
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$RepoOwner,
+
+  [Parameter(Mandatory = $true)]
+  [string]$RepoName,
+
+  [Parameter(Mandatory = $true)]
+  [string]$IssueNumber,
+
+  [Parameter(Mandatory = $true)]
+  [string]$Labels,
+
+  [Parameter(Mandatory = $true)]
+  [string]$AuthToken
+)
+
+. "${PSScriptRoot}\common.ps1"
+
+try {
+  AddIssueLabels -RepoOwner $RepoOwner -RepoName $RepoName `
```

Comment:
nit: Why isn't this and the filename following the Verb-Noun convention?

Reply:
You mean it should be `Add-IssueLabels` ?